app/Models/IndependentActivity.php

```diff
@@ -125,6 +125,30 @@ public function subjects()
         return $this->belongsToMany('App\Models\Subject', 'activity_subject', 'id')->withTimestamps();
     }
 
+    /**
+     * insert the independent activity subjects.
+     */
+    public function Addsubjects()
```

Comment:
Should be camelCase i.e "addSubjects" not "Addsubjects".

Reply:
Done